src/main/java/productDependency/CreateOrderPage.java

```diff
@@ -0,0 +1,1481 @@
+	 	 	CreateOrderPage sp = new CreateOrderPage(driver);
+	 	 	log.info("click on dependenct product.");
+	 	 	String DependentProduct = BasePage.getCellData(xlsxName, sheetName, 7, 2);
+	 	 	driver.findElement(By.xpath("//a[not(@href)]/strong[text()='" + DependentProduct + "']")).click();
+	 	 		}
+	 	 		
+	 //@FindBy(how=How.XPATH, using="//*[@id="dependencies-products-change_-3"]/tbody/tr[3]/td[1]/a/strong
+	 private WebElement clickDependentProduct8;
+
+	 	 		/**
+	 	 		 * Method to click on dependent product.
+	 	 		 * 
+	 	 		 * @throws IOException
+	 	 		 */
+	 public void clickDependentProduct8() throws IOException {
+	 	 	CreateOrderPage sp = new CreateOrderPage(driver);
+	 	 	log.info("click on dependenct product.");
+	 	 	String DependentProduct = BasePage.getCellData(xlsxName, sheetName, 7, 0);
+	 	 	driver.findElement(By.xpath("//a[not(@href)]/strong[text()='" + DependentProduct + "']")).click();
+	 	 		}
+	 
+	 	
+	 	
+	 
+	 //@FindBy(how=How.XPATH, using="//*[@id="dependencies-products-change_-3"]/tbody/tr[3]/td[1]/a/strong
+	 private WebElement clickDependentProduct9;
+
+	 	 		/**
+	 	 		 * Method to click on dependent product.
+	 	 		 * 
+	 	 		 * @throws IOException
+	 	 		 */
+	 public void clickDependentProduct9() throws IOException {
+	 	 	CreateOrderPage sp = new CreateOrderPage(driver);
+	 	 	log.info("click on dependenct product.");
+	 	 	String DependentProduct = BasePage.getCellData(xlsxName, sheetName, 7, 1);
+	 	 	driver.findElement(By.xpath("//a[not(@href)]/strong[text()='" + DependentProduct + "']")).click();
+	 	 		}
+	 
+	 //@FindBy(how=How.XPATH, using="//*[@id="dependencies-products-change_-3"]/tbody/tr[3]/td[1]/a/strong
+	 	private WebElement clickDependentProduct10;
+
+		 	 		/**
+		 	 		 * Method to click on dependent product.
+		 	 		 * 
+		 	 		 * @throws IOException
+		 	 		 */
+		 public void clickDependentProduct10() throws IOException {
+		 	 	CreateOrderPage sp = new CreateOrderPage(driver);
+		 	 	log.info("click on dependenct product.");
+		 	 	String DependentProduct = BasePage.getCellData(xlsxName, sheetName, 7, 2);
+		 	 	driver.findElement(By.xpath("//a[not(@href)]/strong[text()='" + DependentProduct + "']")).click();
+		 	 		}
+		
+	 @FindBy(how = How.XPATH, using = "//button[@id='currentOrder']")
+	//*[@id="currentOrder"]
+	 private WebElement clickCurrentOrderButton;
+
+	/**
+	 * Method to click on Current Order button.
+	 * 
+	 * @throws IOException
+	 */
+	 public void clickCurrentOrderButton() throws IOException {
+		CreateOrderPage sp = new CreateOrderPage(driver);
+		log.info("click on Current Order button.");
+		Assert.assertTrue(clickCurrentOrderButton.isDisplayed());
+		clickCurrentOrderButton.click();
+		JavaScriptExec.sleep();
+	}
+	
+	
+	@FindBy(how = How.XPATH, using = "//*[@id='change--5.quantityAsDecimal']")
+	//*[@id="change--5.quantityAsDecimal"]
+	private WebElement enterQuantity;
+
+	/**
+	 * Method to Enter Price Rate.
```

Comment:
@mayankchhajer Please remove the unused xpath

Reply:
Done